modules/behavior/select.js

```diff
@@ -23,6 +23,12 @@ export function behaviorSelect(context) {
 
 
     function click() {
+        var rtClick = d3.event.type === 'contextmenu';   
```

Comment:
`contextmenu` is not _quite_ right click.  On a Mac it fires on `mousedown`, and on Windows it fires on `mouseup`. 

The code here might be totally ok, this is just a reminder to myself to check the reselection behavior on Windows.